operational_analysis/methods/plant_analysis.py

```diff
@@ -841,7 +841,10 @@ def run_AEP_monte_carlo(self):
             
             # Annual values of lt gross energy, needed for IAV
             reg_inputs_lt['gross_lt'] = gross_lt
-            gross_lt_annual = reg_inputs_lt['gross_lt'].resample('12MS').sum().values            
+            gross_lt_annual = np.zeros(self._run.num_years_windiness)
+            # Annual resample starting on the first day in reg_inputs_lt
+            for j in np.arange(self._run.num_years_windiness):
+                gross_lt_annual[j] = reg_inputs_lt['gross_lt'].loc[(reg_inputs_lt.index >= reg_inputs_lt.index[0] + j*pd.tseries.offsets.DateOffset(months = 12)) & (reg_inputs_lt.index < reg_inputs_lt.index[0] + (j+1)*pd.tseries.offsets.DateOffset(months = 12))].sum()
```

Comment:
This looks like it is working as expected, although I wonder if there is a more elegant way to compute this without a for loop. One thought is could you still use the resample('12MS') method but first shift the time index so that it starts at the beginning of the month? Maybe @jordanperr has some ideas here.

Reply:
Here is another method you could consider that seems to work correctly. It avoids a for loop, but still involves a lot of code:
```
reg_inputs_lt_temp = reg_inputs_lt['gross_lt']
# now shift time index to beginning of first month so resampling by 'MS' works as expected
reg_inputs_lt_temp.index = reg_inputs_lt_temp.index - pd.Timedelta(reg_inputs_lt_temp.index[0] - (reg_inputs_lt_temp.index[0].floor('d') + pd.offsets.MonthEnd(0) - pd.offsets.MonthBegin(1)))
gross_lt_annual = reg_inputs_lt_temp.resample('12MS').count().values
```

Reply:
Either way, these long lines should be split into multiple lines to adhere to the 120 character linewdith

Reply:
I think that this functionality could be its own function in the `toolkits/timeseries.py` module with caching because we're going to recompute the same thing a number of times as we simulate this, but if we don't want to go that route, the solution you have Eric is what I'd recommend. I tried to get really slick about it, but pandas doesn't want to play nice in a way that makes things easy and clean. That said, I'd break it up into lines like the following (adjust for proper indents after copy/paste):
```python
reg_inputs_lt_temp = reg_inputs_lt['gross_lt']
# now shift time index to beginning of first month so resampling by 'MS' works as expected
reg_inputs_lt_temp.index = (
    reg_inputs_lt_temp.index = (
    reg_inputs_lt_temp.index - pd.Timedelta(
        reg_inputs_lt_temp.index[0]
        - (
            reg_inputs_lt_temp.index[0].floor('d')
            + pd.offsets.MonthEnd(0)
            - pd.offsets.MonthBegin(1))
        )
)
)
gross_lt_annual = reg_inputs_lt_temp.resample('12MS').sum().values
```

Reply:
@RHammond2 and @nbodini I made a separate function `get_annual_values` to calculate the annual values of LT gross energy and use this in the AEP analysis now. Please take a look and see what you think.

Reply:
Looks good to me! And thanks for doing this.